### PR TITLE
export PYTHON to make process

### PR DESCRIPTION
**Pull request by paaguti**
I finally left this... the ryu install process may need a couple more retouches. Did you know that Faucet took it over and that the "new" github is faucetsdn/ryu?

### CODE REVIEW

**Comment by lantz** [comment]
Haha, yes I did, but I seem to have forgotten that it would impact mininet's `install.sh`.

File: util/install.sh [hunk 1]
```diff
@@ -180,7 +180,7 @@ function mn_deps {
 
     echo "Installing Mininet core"
     pushd $MININET_DIR/mininet
-    sudo PYTHON=${PYTHON} make install
+    sudo PYTHON=${PYTHON} make PYTHON=${PYTHON} install
```
Comment (lantz) [inline util/install.sh:183]: So I just tested this, and it appears that 
    sudo PYTHON=${PYTHON} make install
is fine. Do your tests indicate otherwise?

**Comment by paaguti** [comment]
Therese are a couple quirks more.... Give me couple mins to switch computers...

/PA

Enviado desde mi iPhone

File: util/install.sh (hunk 1 above)
Comment (paaguti) [reply util/install.sh:183]: Hi,

To check this and faucetsdn pull request, I have made a 
```util/install.sh -3nvfy```
on a clean Ubuntu 20.04 VM with the Mate desktop and I have a running mininet with an updated Ryu controller

Best, /PA

**Comment by paaguti** [comment]
So this is my current status, after testing with a 
`util/install.sh -nfvy` 
on the Ubuntu focal VM I'm going to use for my courses. It appears that focal, now a couple of months into its life, is really picky with the Python stuff.

**Comment by lantz** [comment]
20.04 install is fixed in 2.3.0.

**Pull request closed without merging**